deployments/data100/config/common.yaml

```diff
@@ -28,7 +28,7 @@ jupyterhub:
           - access:servers!group=course::1527426
         # this role will be assigned to...
         groups:
-          - Admins::1527426 #not fully sure if this number should be the course code (1527426) or the bcourses group code (383808)
+          - course::1527426::group::Admins #not fully sure if this number should be the course code (1527426) or the bcourses group code (383808)
```

Comment:
when we're sure that the PR is ready2merge, could we delete the comment at the end of the line?  :)